nipype/interfaces/fsl/epi.py

```diff
@@ -523,11 +521,27 @@ def _num_threads_update(self):
             self.inputs.environ['OMP_NUM_THREADS'] = str(
                 self.inputs.num_threads)
 
-    def _use_cuda(self):
-        if self.inputs.use_cuda:
-            _cmd = 'eddy_cuda'
-        else:
-            _cmd = 'eddy_openmp'
+    @staticmethod
+    def _eddycmd(use_cuda):
+        logger = logging.getLogger('interface')
+        if 'FSLDIR' not in os.environ:
+            logger.warn("FSLDIR not set: assuming command 'eddy'")
+            return 'eddy'
+
+        FSLDIR = os.environ['FSLDIR']
+        if use_cuda and os.path.exists(os.path.join(FSLDIR, 'eddy_cuda')):
```

Comment:
shouldn't the `and` clause be on a separate line? otherwise if cuda is not found it will default to one of openmp or plain eddy.